codec-http/src/main/java/io/netty/handler/codec/http/QueryStringDecoder.java

```diff
@@ -224,6 +280,10 @@ private void decodeParams(String s) {
                     name = decodeComponent(s.substring(pos, i), charset);
                 }
                 pos = i + 1;
+            } else if (withCommaDecoding && c == ',' && name != null) {
```

Comment:
Actually basically I agree with slandelle. I mean your case would be a edge case and I mean what about `a,b,c=y,x,z` I guess that would need to be a lot of more work to actually end up with `a=y a=x a=z b=y b=x b=z c=y c=x c=z` and I guess it's not really worth it. I actually thought that only doing it in value's is the way to go.